js/fullPictures.js

```diff
@@ -0,0 +1,68 @@
+import {newTestData} from './data.js';
+
+const currentPictures = document.querySelectorAll ('.picture');
+const modalPicture = document.querySelector('.big-picture');
+const commentsList = document.querySelector ('.social__comments');
+const newCommentTemplate = document.querySelector ('#comment').content;
+
+const addEscHandler = function (el) {
+  addEventListener('keydown', (event) => {
+    const key = event.key;
+    if (key === 'Escape') {
+      el.classList.add('hidden');}
+  });
+};
+
+const expandPicture = function (el) {
```

Comment:
```suggestion
const expandPicture = (el) => {
```
у тебя где-то стрелочные функции, а где-то обычные. должно быть единообразно